apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-component.md

```diff
@@ -0,0 +1,290 @@
+---
+title: 'Tutorial: Create your first XM Cloud no-code component'
+description: 'In this tutorial, we will go through the steps to create a Text Teaser component with a no-code approach using XM Cloud Component builder'
+openGraphImage: 'https://sitecorecontenthub.stylelabs.cloud/api/public/content/21dabc30da2c475a8549640a04885a46?v=18b721db'
+---
+
+## Introduction - What You are Going to Learn
+
+In this tutorial, we will go through the steps to create a Text Teaser component with a no-code approach using XM Cloud Component builder. The walkthrough will not show all possible options, but will take you through a flow to define and lay out a new Component to use on your home page. You will learn:
```

Comment:
I would say XM Cloud Component Builder --> with capital B

Reply:
also in other occasions. Not marking all of them now.

Reply:
I'm not sure this is the official product naming for that feature. If you reference the docs: https://doc.sitecore.com/xmc/en/users/xm-cloud/build-components.html

They use the term "XM Cloud Components" and the "Components builder". I still need to make a change because we have been referring to it as Component builder and not Components builder. Is there a product marketing change to this feature name such that we should now capitalize the B?

Reply:
I've pushed up the change to rename everything to XM Cloud Components, XM Cloud Components builder, or Components builder to match with the docs.

If we do need to change to 'Builder', let's double-check with product marketing and align with docs so we get that change everywhere.